lib/jose-jwe-decrypt.js

```diff
@@ -19,11 +19,15 @@
  *
  * @param cryptographer  an instance of WebCryptographer (or equivalent). Keep
  *                       in mind that decryption mutates the cryptographer.
- * @param key_promise    Promise<CryptoKey>, either RSA or shared key
+ * @param key    CryptoKey, Promise<CryptoKey>, either RSA or shared key
```

Comment:
if I understand correctly, you want the api to take a CryptoKey or a Promise<CryptoKey>?